README.md

```diff
@@ -241,7 +241,9 @@ You can also change the country e.g. `sold_in_uk=1`, `sold_in_de=1`, etc.
   * [Bills](http://api.parliamentdata.ca/bills)
   * [Representatives](http://api.parliamentdata.ca/representatives)
   * [Committees](http://api.parliamentdata.ca/committees)
-
+* Scottish Parliament
+  * [Government Roles](https://data.parliament.scot/api/governmentroles)
+  
```

Comment:
@jdalrymple There seems to be a trailing space on line `246` which is causing the test to fail. Please remove the trailing space, commit and push. =)